.travis.yml

```diff
@@ -28,8 +28,10 @@ before_cache:
 
 matrix:
   include:
-    - compiler: "ghc-8.6.1"
+    - compiler: "ghc-8.8.3"
```

Comment:
I'm surprised this was the only version added by haskell-ci, but I'll see what the version specifier in the cabal file looks like.

Reply:
If I paid a little more attention to your comment. I wouldn't need to debug the .travis.yml by hand. :S